docs/blog/v5-intro.md

```diff
@@ -0,0 +1,167 @@
+kind: Grafana
+metadata:
+  name: grafana
+  labels:
+    dashboards: "grafana"
+spec:
+  config:
+    log:
+      mode: "console"
+    auth:
+      disable_login_form: "false"
+    security:
+      admin_user: root
+      admin_password: secret
+```
+
+And the grafana dashboard with the instanceSelector, the same thing goes for the other grafana resources.
+
+```.yaml
+apiVersion: grafana.integreatly.org/v1beta1
+kind: GrafanaDashboard
+metadata:
+  name: grafanadashboard-sample
+spec:
+  resyncPeriod: 30s
+  instanceSelector:
+    matchLabels:
+      dashboards: "grafana"
+  json: >
+    {
+      "id": null,
+      "title": "Simple Dashboard",
+      "tags": [],
+      "style": "dark",
+      "timezone": "browser",
+      "editable": true,
+      "hideControls": false,
+      "graphTooltip": 1,
+      "panels": [],
+      "time": {
+        "from": "now-6h",
+        "to": "now"
+      },
+      "timepicker": {
+        "time_options": [],
+        "refresh_intervals": []
+      },
+      "templating": {
+        "list": []
+      },
+      "annotations": {
+        "list": []
+      },
+      "refresh": "5s",
+      "schemaVersion": 17,
+      "version": 0,
+      "links": []
+    }
+```
+
+## New Features
+
+This is probably the part of the blog that you are most exited about, it goes through a number of the new features that we support.
+
+### Multi grafana instances support
+
+This is one of our most requested and [oldest issues](https://github.com/grafana-operator/grafana-operator/issues/174) in the grafana-operator and we have finally been able to provide it.
+
+In v5 you can now run multiple instances of grafana in the same namespace and externally.
+
+### External grafana instances
+
+It's now possible to configure external grafana instances through the operator.
+So if you are using grafana cloud or a hosted grafana instances in one of the big cloud providers you can still use gitops to apply your dashboards to them.
+
+One of the limitations with the external grafana instances is that we don't support installing plugins.
+If your dashboard is using a plugin we assume that you have manually installed it to your grafana instance.
+
+### Grafana config
+
+The deployment configuration is more **opinionated** in it's default setup while at the same time giving you the user access to the whole deployment spec so you can overwrite anything you want.
+For example when you start a grafana instance it will by default have decent security defaults, it will set `RunAsNonRoot` on your grafana instance when they get created.
+But since you have access to the whole deployment spec you can always overwrite these kind of settings.
+
+The same thing applies to all our resources, for example ingress.
+The bad thing here is of course that we won't hide any of the logic that made the operator "easy" to get started with.
+So if you want to configure an ingress for your grafana instance you will have to define it just like any other ingress resources.
```

Comment:
Refer to an example for this, like the normal nginx one and openshift route?

Reply:
I don't want to many example in this document, I added a ingress-nginx one. 
Instead I will focus on examples in the getting-started part.